docker-compose.prod.yml

```diff
@@ -52,10 +52,21 @@ services:
       - QUERY_LIST_SORTABLE=true
     command: "foreman s -f Procfile"
     ports:
-      - 80:3000 # Map to port 80 for outside users.
+      - 3000:3000 # Map to port 80 for outside users when you are not using Nginx
```

Comment:
I wonder if exposing port 3000 even makes sense in Prod, when you have nginx?????   Would it make sense to leave the port mapping as `80:3000`, but comment it out, as well as have the comment you added?